src/block_cache/block_tier.c

```diff
@@ -166,3 +179,14 @@ __wt_blkcache_get_handle(WT_SESSION_IMPL *session, WT_BM *bm, uint32_t objectid,
 
     return (ret);
 }
+
+/*
+ * __wt_blkcache_handle_read_done --
+ *     Update block handle when a read operation completes.
+ */
+void
+__wt_blkcache_handle_read_done(WT_SESSION_IMPL *session, WT_BLOCK *block)
```

Comment:
Minor naming nit: We have `session_get_dhandle` and `session_release_dhandle`. I think this should follow the same naming convention and be more generic. Currently it only matters for read operations, but the work to get and release may change in the future.

Reply:
Yes. That's definitely better. Changed.